### DISCUSSION TITLE
Unable to connect to Rustdesk client on the self-hosted server

### DISCUSSION THREAD

**Discussion by Road-To-Gensokyo** [rustdesk/rustdesk#9521] [Q&A]
I have deployed a Rustdesk server and client on a Windows server, other computers can connect each other normally but I can't connect to the client on the server unless I enable dmz forwarding.
我在一台win服务器上部署了rustdesk服务以及客户端，其他电脑可以正常使用但是除非打开dmz转发否则无法连接到服务器上的rustdesk客户端

**Comment by rustdesk** [comment]
#763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with self-hosting here.
